WorkloadManagementSystem/JobWrapper/test/Test_JobWrapper.py

```diff
@@ -66,7 +66,7 @@ def test_InputData( self ):
     self.assertTrue( res['OK'] )
 
   def test__performChecks( self ):
-    wd = WatchdogLinux( os.getpid(), MagicMock(), MagicMock(), 1000, 1024 * 1024 )
+    wd = WatchdogLinux( os.getpid(), MagicMock(), MagicMock(), 1000, 1024 * 1024, jobArgs = { 'StopSigNumber' : 10 } )
```

Comment:
as  well..

Reply:
Thanks: I've made all the Watchdog*** calls use named arguments now and made the argument names consistent in the signatures (thread vs exeThread)